### PR TITLE
Combine back and delete button into dropdown menu in feature panel header.

### LINKED ISSUE #711: [Delete feature] Delete feature option gone
Introduced in #451. Can we add this back to the overflow menu? 

@jacobmclaws Did you have mocks for this?

### PR DESCRIPTION
**Pull request by DaoyuT**
Screenshot:
Screen Shot 2021-06-21 at 3 09 30 PM

Fixes #711.

### CODE REVIEW

**Comment by gino-m** [comment]
> Screenshot:
> Screen Shot 2021-06-21 at 3 09 30 PM

Would you like to put the "x" next to the overflow menu for now like in the example you showed me in Contacts? It seems odd to have the back/close option in an overflow. Wdyt?

**Comment by gino-m** [comment]
> Screenshot:
> Screen Shot 2021-06-21 at 3 09 30 PM
> 
> Fixes #711.

@jacobmclaws Do you have any suggestions on how to present a header (eventually multiline) with an overflow and back/close CTA?

**Comment by jacobmclaws** [comment]
I agree it's quite strange to have the button that takes you back in the dropdown. I would suggest a button above with some text. Something like this:
image

Not sure that "layers list" is the appropriate name for that side-panel view. Just "Back" probably also works in this context. I would suggest using our Ground green color for it to indicate it's part of the navigational system UI. Let me know if that makes sense. Thanks!

**Comment by DaoyuT** [comment]
Done. Thanks @jacobmclaws for the suggestion.
Screen Shot 2021-07-05 at 6 16 42 PM

File: web-ng/src/app/components/feature-panel-header/feature-panel-header.component.html [hunk 1]
```diff
@@ -14,27 +14,25 @@
 limitations under the License.
 -->
 
+<button mat-button
+        color="primary"
+        aria-label="back"
+        (click)="onCloseClick()">
+  <mat-icon>arrow_back</mat-icon>
+  <span>Back to layers list</span>
```
Comment (gino-m) [inline web-ng/src/app/components/feature-panel-header/feature-panel-header.component.html:22]: Please replace with "Back" as suggested by @jacobmclaws

**Review by gino-m — APPROVED** [review]
One comment, otherwise LGTM.

**Review by gino-m — APPROVED** [review]
Made the change directly. LGTM!

**Pull request merged**